pkg/controller/vsphere/reconciler.go

```diff
@@ -442,6 +442,11 @@ func clone(s *machineScope) (string, error) {
 
 	resourcepool, err := s.GetSession().Finder.ResourcePoolOrDefault(s, resourcepoolPath)
 	if err != nil {
+		var multipleFoundError *find.MultipleFoundError
```

Comment:
before we return an error, as far as I can see we keep returning an error? How does this make any difference for the machine to be stuck provisioning?

Reply:
was this mean to return `InvalidMachineConfiguration()`?

Reply:
you are right, changed it to `InvalidMachineConfiguration()`